### ISSUE TITLE
Fix build removing usage of itkMultiplyByConstantImageFilter

### ISSUE THREAD

**Issue by jcfr** [tokjun/ErodeDilateLabel#2]
Usage of `itkMultiplyByConstantImageFilter` prevents the extension from building.

```
ErodeDilateLabel/ErodeLabel/ErodeLabel.cxx:35:46: fatal error: itkMultiplyByConstantImageFilter.h: No such file or directory
```

Error similar to
* https://github.com/ljzhu/LASegmenter/issues/4
* https://github.com/gaoyi/SegmentationAidedRegistration/issues/7

**Comment by jcfr** [comment]
Good news. This was fixed for `LASegmenter` by @blowekamp. See https://github.com/ljzhu/LASegmenter/pull/3

**Comment by jcfr** [comment]
Closing. This was fixed in #1

**Comment by jcfr** [comment]
Slicer ExtensionsIndex was updated in https://github.com/Slicer/ExtensionsIndex/pull/1592
